circle.yml

```diff
@@ -127,3 +130,4 @@ test:
       unicode/utf16
       unicode/utf8
     - go test -v -race ./...
+    - gopherjs test -v --minify fmt # Minification should work.
```

Comment:
See comment above.

Running with `--minify` on the standard library increases the test surface area for minified code which is a good thing.